packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -188,26 +212,110 @@ const OrderDetails = observer(({ onPageReturn }) => {
+                                    <UserRatingButton
+                                        button_text={
+                                            is_reviewable ? localize('Rate this transaction') : localize('Not rated')
+                                        }
+                                        is_disabled={!is_reviewable}
+                                        large
+                                        onClick={() => order_store.setIsRatingModalOpen(true)}
+                                    />
+                                </div>
+                                <Text color='less-prominent' size='xxxs'>
+                                    {!is_reviewable && (
+                                        <Localize i18n_default_text='You can no longer rate this transaction.' />
+                                    )}
                                 </Text>
-                            }
-                            value={contact_info || '-'}
-                        />
-                        <MyProfileSeparatorContainer.Line className='order-details-card--line' />
-                        <OrderInfoBlock
-                            className='order-details-card--padding'
-                            label={
-                                <Text size='xs' weight='bold'>
-                                    {labels.instructions}
+                            </React.Fragment>
+                        )}
+                        {review_details && (
+                            <div className='order-details-card__ratings'>
+                                <Text color='prominent' size='s' weight='bold'>
+                                    <Localize i18n_default_text='Your transaction experience' />
                                 </Text>
-                            }
-                            value={advert_details.description.trim() || '-'}
-                        />
+                                <div className='order-details-card__ratings--row'>
+                                    <StarRating
+                                        empty_star_className='order-details-card__star'
+                                        empty_star_icon='IcEmptyStar'
+                                        full_star_className='order-details-card__star'
+                                        full_star_icon='IcFullStar'
+                                        initial_value={rating_average_decimal}
+                                        is_readonly
+                                        number_of_stars={5}
+                                        should_allow_hover_effect={false}
+                                        star_size={isMobile() ? 17 : 20}
+                                    />
+                                    <div className='order-details-card__ratings--row'>
+                                        {review_details.recommended === 1 ? (
+                                            <>
```

Comment:
Shouldn't we wrap this with React.Fragment as it was used throughout this file, for consistency

Reply:
Yes, absolutely 👍